performance/benchmark.rb

```diff
@@ -4,14 +4,19 @@
 Liquid::Template.error_mode = ARGV.first.to_sym if ARGV.first
 profiler = ThemeRunner.new
 
+# this profiler is for benchmarking render only
+profiler_render = ThemeRunner.new
+profiler_render.compile_tests
```

Comment:
ok, so I see what you're getting at here: there are two phases, parse/compile (A) and run/render (B). It does feel weird that our benchmarks here are {A, A+B}. It does make more sense to have {A, B} or {A, B, A+B}.

I don't see any reason this `profiler_render` has to be a separate variable. Could we just add `profiler.compile_tests` after line 5? Or better yet, just call `compile_tests` from `ThemeRunner#initialize`?